regparser/web/jobs/urls.py

```diff
@@ -3,8 +3,19 @@
 from rest_framework.urlpatterns import format_suffix_patterns
 
 urlpatterns = [
-    url(r'^job/$', views.JobViewList.as_view()),
-    url(r'^job/(?P<job_id>[-a-z0-9]+)/$',
+    url(r'^job(/)$', views.JobViewList.as_view()),
```

Comment:
Let's work through that at some point. I'm betting this is a regex confusion. Not worth holding up the PR, though.